tests/OSPSuite.SimModel.Tests/SimulationSpecs.cs

```diff
@@ -481,6 +481,7 @@ public void should_load_finalize_and_run_simulation(string shortFileName)
 
       protected static IEnumerable<string> TestData()
       {
+         yield return "12.0_Brockmoller2005_CPA";
```

Comment:
one of the simulation from the HI qualification which produce negative values in V12